cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/v7/ConsentStatusApiModel.kt

```diff
@@ -0,0 +1,95 @@
+package com.sourcepoint.cmplibrary.data.network.model.v7
+
+import com.sourcepoint.cmplibrary.data.network.util.Env
+import com.sourcepoint.cmplibrary.model.exposed.CcpaStatus
+import com.sourcepoint.cmplibrary.model.exposed.GDPRPurposeGrants
+import org.json.JSONObject
+
+internal class ConsentStatusResp(
+    val thisContent: JSONObject,
+    val consentStatusData: ConsentStatusData,
+    val localState: JSONObject
+)
+
+internal data class ConsentStatusParamReq(
+    val env: Env,
+    val metadata: String,
+    val propertyId: Int,
+    val accountId: Int,
+    val authId: String? = null
+)
+
+internal data class ConsentStatusData(
+    val thisContent: JSONObject,
+    val gdprCS: GdprCS?,
+    val ccpaCS: CcpaCS?
+)
+
+internal data class GdprCS(
+    val thisContent: JSONObject,
+    var grants: Map<String, GDPRPurposeGrants> = emptyMap(),
+    val euconsent: String,
+    val addtlConsent: String,
+    val dateCreated: String,
+    val consentUUID: String,
```

Comment:
I believe `consentUUID` and `ccpaUUID` can be ignored since they are duplicates of the attribute `uuid` .

Reply:
sure, I can delete them